x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -305,15 +306,23 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
             wroteWarningStatus = true;
           }
 
-          const { tuples, remainingGap } = getRuleRangeTuples({
+          const {
+            tuples,
+            remainingGap,
+            wroteWarningStatus: rangeTuplesWarningStatus,
+          } = await getRuleRangeTuples({
```

Comment:
This method should also return a warning message and assign it to the `warningMessage` variable.